src/Private/Authentication.ps1

```diff
@@ -1068,31 +1068,85 @@ function Invoke-PodeAuthValidation {
     # if it's a merged auth, re-call this function and check against "succeed" value
     if ($auth.Merged) {
         $results = @{}
+        if ([version]$PSVersionTable.PSVersion -ge [version]'7.0.0') { 
+            # Run the foreach loop in parallel
+            $auth.Authentications | ForEach-Object -Parallel {
```

Comment:
This should be it's own workstream/ticket, not a part of OpenAPI. Also, this logic was never designed to run in parallel, it has to be sequential due tot eh way certain auth methods work. Additional, `-Parallel` on `Foreach` is super slow, so would cause a performance issue on authentication requests.

Please remove this commit (a part from the LICENSE.txt changes).

Reply:
ok

this change solve the issue with multiple auth hung when OAUTH server is not reachable

Reply:
Sounds like a bug I could look into if you want to raise a bug ticket for it? I know that having OAuth2 can cause some quirkiness, and is useless best placed as the last possible method to try due to all the redirecting involved.